testDokka.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
```

Comment:
Exit when any command fails?
 
 ```suggestion
set -e
```

Reply:
This is not convenient because it stops webserver after the script is finished